File: CRS-Signify-POS-Group-A/src/com/signify/bean/Student.java
package com.signify.bean;


public class Student extends User{
	private int age;
	private int isApproved;
	private String studentid, fname, bloodgroup, phnum, name, branch;
	
	public int getIsApproved() {
		return isApproved;
	}
	public void setIsApproved(int isApproved) {
		this.isApproved = isApproved;
	}
	public String getBranch() {
		return branch;
	}
	public void setBranch(String branch) {
		this.branch = branch;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
	public String getStudentid() {
		return studentid;
	}
	public void setStudentid(String studentid) {
		this.studentid = studentid;
	}
	public String getFname() {
		return fname;
	}
	public void setFname(String fname) {
		this.fname = fname;
	}
	public String getBloodgroup() {
		return bloodgroup;
	}
	public void setBloodgroup(String bloodgroup) {
		this.bloodgroup = bloodgroup;
	}
	public String getPhnum() {
		return phnum;
	}
	public void setPhnum(String phnum) {
		this.phnum = phnum;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int isApproved() {
		return isApproved;
	}
	public void setApproved(int isApproved) {
		this.isApproved = isApproved;
	}
	public String getStudentId() {
		return studentid;
	}
	public void setStudentId(String studentId) {
		this.studentid = studentId;
	}	
	
}

File: CRS-Signify-UI-UX-Group-A/src/app/model/student.ts
import { User } from "./user";

export class Student extends User {
    public studentid:string;
    public age:number;
    public fname:string;
    public bloodgroup:string;
    public phnum:string;
    public branch:string;


    constructor(name:string,userId:string,address:string,password:string,doj:string,studentid:string,age:number,fname:string,bloodgroup:string,phnum:string,branch:string){
        super(name, userId, address, password, doj);
        this.studentid=studentid;
        this.age=age;
        this.fname=fname;   
        this.bloodgroup=bloodgroup;
        this.phnum=phnum;
        this.branch=branch;



    }
}
    
